qiskit/optimization/converters/integer_to_binary.py

```diff
@@ -188,40 +188,48 @@ def _substitute_int_var(self):
 
         # set linear constraints
         for constraint in self._src.linear_constraints:
-            linear, constant = self._encode_linear_coefficients_dict(constraint.linear.to_dict())
-            self._dst.linear_constraint(linear, constraint.sense,
-                                        constraint.rhs - constant, constraint.name)
+            linear, constant = self._convert_linear_coefficients_dict(constraint.linear.to_dict())
+            self._dst.linear_constraint(
+                linear, constraint.sense, constraint.rhs - constant, constraint.name
+            )
 
         # set quadratic constraints
         for constraint in self._src.quadratic_constraints:
-            linear, linear_constant = self._encode_linear_coefficients_dict(
-                constraint.linear.to_dict())
-            quadratic, quadratic_linear, quadratic_constant = \
-                self._encode_quadratic_coefficients_dict(constraint.quadratic.to_dict())
-
-            constant = linear_constant + quadratic_constant
-            for i, v in quadratic_linear.items():
+            linear, linear_constant = self._convert_linear_coefficients_dict(
+                constraint.linear.to_dict()
+            )
+            quadratic, q_linear, q_constant = self._convert_quadratic_coefficients_dict(
+                constraint.quadratic.to_dict()
+            )
+
+            constant = linear_constant + q_constant
+            for i, v in q_linear.items():
                 linear[i] = linear.get(i, 0) + v
 
-            self._dst.quadratic_constraint(linear, quadratic, constraint.sense,
-                                           constraint.rhs - constant, constraint.name)
+            self._dst.quadratic_constraint(
+                linear, quadratic, constraint.sense, constraint.rhs - constant, constraint.name
+            )
 
-    def decode(self, result: OptimizationResult) -> OptimizationResult:
-        """Convert the encoded problem (binary variables) back to the original (integer variables).
+    def interpret(self, result: OptimizationResult) -> OptimizationResult:
+        """
+        Convert back the converted problem (binary variables) to the original (integer variables).
```

Comment:
The documentation guidelines are w/o linebreak here

```suggestion
        """Convert back the converted problem (binary variables) to the original (integer variables).
```